File: src/main/java/com/hedvig/backoffice/services/payments/dto/TransactionDTO.java
package com.hedvig.backoffice.services.payments.dto;

import java.time.Instant;
import java.util.UUID;
import javax.money.MonetaryAmount;
import lombok.Value;

@Value
public class TransactionDTO {
  UUID id;
  MonetaryAmount amount;
  Instant timestamp;
  String transactionType;
  String transactionStatus;
}

File: src/main/java/com/hedvig/backoffice/services/payments/dto/TransactionDTO.kt
package com.hedvig.backoffice.services.payments.dto

import java.time.Instant
import java.util.*
import javax.money.MonetaryAmount

data class TransactionDTO(
  var id: UUID,
  var amount: MonetaryAmount,
  var timestamp: Instant,
  var transactionType: String,
  var transactionStatus: String
)
